app/controllers/api/payment/braintree_controller.rb

```diff
@@ -29,15 +29,24 @@ def webhook
     render json: {success: true}
   end
 
+  def delete_payment_method
+    # TODO: Delete payment method on Vault, and if that's successful, destroy it locally.
```

Comment:
that makes sense but I don't think we should deploy it because it's unused and presents a public API onto messing with our db